README.md

```diff
@@ -503,3 +504,16 @@ total_failures = wavefront_sender.get_failure_count()
   the minor and patch versions.
 * Reach out to us on our public [Slack channel](https://www.wavefront.com/join-public-slack).
 * If you run into any issues, let us know by creating a GitHub issue.
+
+## How to Release
+
+1. Make sure that all changes for the release are merged with the master branch.
+2. Open the `setup.py` file from the top level directory of the project.
+3. Search for `version=` in the file. There you will find the version number for example `1.8.10`.
+4. Log into Github, click on **Releases** in the about section on the right, and click on **Draft a new release.**
+5. For **version**, choose the version you found in step 3, but prefix it with `v` for example `v1.8.10
+6. Provide a short but descriptive title for the release.
+7. Fill in the details of the release. Please copy the markdown from the previous release and follow the same format.
+8. Click on **Publish release.**. This will automatically start an action that will publish the the release to pypi.org.
+9. From the Github top navigation bar of this project, click on the **Actions** tab. On the first line of the list of workflows, you should see a workflow running which will publish your release to pypi.org
+10. When the workflow from step 9 has a green checkmark by it, go to [pypi.org](https://pypi.org/project/wavefront-sdk-python/) to verify that the latest version has been released.
```

Comment:
When the workflow from step 9 has a green checkmark next to it, go to [pypi.org](https://pypi.org/project/wavefront-sdk-python/) and verify that the latest version is published.

Reply:
Done.